react-refresher/my-app/src/App.js

```diff
@@ -0,0 +1,13 @@
+import './App.css';
+import Counter from './Counter';
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <Counter initialNum="0" />
```

Comment:
What you are doing works, and you were careful to convert the `initialNum` prop to a Number in the Counter component. However, usually when you pass numerical values in React, you use `{0}` instead of `"0"`. The same holds for other non-string values, like booleans.